api/tests/conftest.py

```diff
@@ -54,11 +55,12 @@ def new_location():
     return location
 
 
-@pytest.fixture()
+@pytest.fixture(scope='session')
 def app():
     app = create_app('test')
     with app.app_context():
         db.create_all()
         yield app
         db.session.remove()
         db.drop_all()
+    reset('dev')
```

Comment:
Do we want ```reset``` to be called with ```'test'```?

Reply:
This was a stopgap to deal with the tests erasing the dev db. It just made it so you didn't have to manually reset the dev db after running tests. But now that I got a separate test db working I will remove that since it would prevent the dev db from persisting across tests. I'll push a commit this morning that will remove it.

Reply:
My last commit removed this from the test fixture. Seems to work so now the tests aren't resetting the dev db or doing anything else to the dev db as far as I can tell. I tested it by modifying the dev db to see if it persisted across tests and seems to be working as expected.